third_party/blink/renderer/includes.gni

```diff
@@ -6,6 +6,8 @@ brave_blink_renderer_core_visibility =
 brave_blink_renderer_core_public_deps = [ "//brave/third_party/blink/renderer" ]
 
 brave_blink_renderer_core_sources = [
+  "//brave/third_party/blink/renderer/core/brave_session_cache.cc",
```

Comment:
I think `//brave/third_party/blink` should actually be removed (not as part of this PR). I think it's confusing third party deps for brave-core with upstream overrides. For now put these in chromium_src/third_party/blink/...

Reply:
hmm... actually maybe we need a different place for these so just leave them and we can change it all in a follow-up